platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,742 @@
+  - The progress of individual pipeline **Processes**
+  - **Aggregated stats** for the run (total walltime, CPU hours)
+  - **Workflow metrics** (CPU efficiency, memory efficiency)
+  - A **Task details** table for every task in the workflow
+
+  The task details table provides further information on every step in the pipeline, including task statuses and metrics.
+
+  #### Task details
+
+  Select a task in the task table to open the **Task details** dialog. The dialog has three tabs: **About**, **Execution log**, and **Data Explorer**.
+
+  The **About** tab includes:
+
+  1. **Name**: Process name and tag
+  2. **Command**: Task script, defined in the pipeline process
+  3. **Status**: Exit code, task status, and number of attempts
+  4. **Work directory**: Directory where the task was executed
+  5. **Environment**: Environment variables that were supplied to the task
+  6. **Execution time**: Metrics for task submission, start, and completion time
+  7. **Resources requested**: Metrics for the resources requested by the task
+  8. **Resources used**: Metrics for the resources used by the task
+
+  ![Task details window](./quickstart-demo/assets/task-details.gif)
+
+  #### Execution log
+
+  The **Execution log** tab provides a real-time log of the selected task's execution. Task execution and other logs (such as stdout and stderr) are available for download from here, if still available in your compute environment.
+
+  #### Task work directory in Data Explorer
+
+  If a task fails, a good place to begin troubleshooting is the task's work directory.
+
+  Nextflow hash-addresses each task of the pipeline and creates unique directories based on these hashes. Instead of navigating through a bucket on the cloud console or filesystem to find the contents of this directory, use the **Data Explorer** tab in the Task window to view the work directory.
+
+  Data Explorer allows you to view the log files and output files generated for each task in its working directory, directly within Platform. You can view, download, and retrieve the link for these intermediate files in cloud storage from the **Data Explorer** tab to simplify troubleshooting.
+
+  ![Task data explorer](./quickstart-demo/assets/sp-cloud-task-data-explorer.gif)
+
+</details>
+
+## Tertiary analysis with Data Studios
+
+Set up an environment for tertiary analysis of your pipeline data and enable collaboration with other workspace participants. 
+
+**Data Studios** streamlines the process of creating interactive analysis environments for Platform users. With built-in templates for platforms like Jupyter Notebook, RStudio, and VSCode, creating a data studio is as simple as adding and sharing pipelines or datasets.
+
+For the purposes of this guide, we will create an RStudio environment in which to normalize the pipeline output data, perform differential expression analysis, and visualize the data with MDS plots, volcano plots, and a heatmap of the most differentially expressed genes.  
```

Comment:
```suggestion
For the purposes of this guide, we will create an RStudio environment in which to normalize the pipeline output data, perform differential expression analysis, and visualize the data with exploratory plots.  
```
I'm not sure the terms 'MDS' or 'volcano plot' will be obvious to everyone in the target audience.